packaging/python/README.md

```diff
@@ -22,15 +22,23 @@ Logout and log back in to have docker service properly configured.
 We mount local neuron repository inside docker as a volume to preserve any code changed. We can use -v option to mount the local folder as:
 
 ```
-docker run -v /home/user/nrn:/root/nrn -v /home/user/mpt-headers/2.21:/nrnwheel/mpt -it neuronsimulator/neuron_wheel bash
+git clone https://github.com/neuronsimulator/nrn.git
+
+docker run -v $PWD/nrn:/root/nrn -v $PWD/mpt-headers/2.21:/nrnwheel/mpt -it neuronsimulator/neuron_wheel bash
 ```
 
-where `/home/user/nrn` is a neuron repository on the host machine and `/home/user/mpt` is a directory containing MPT MPI headers. We mount those directories inside docker at location `/root/nrn` and `/nrnwheel/mpt` inside the container. The MPT headers in the separate repository as it's not open source library. You can download the headers as:
+where `$PWD/nrn` is a neuron repository on the host machine and `$PWD/mpt-headers` is a directory containing HPE-MPT MPI headers (optional). We mount those directories inside docker at location `/root/nrn` and `/nrnwheel/mpt` inside the container. The MPT headers are optional and maintained in the separate repository as it's not open source library. You can download the headers as:
 
 ```
 git clone ssh://bbpcode.epfl.ch/user/kumbhar/mpt-headers
 ```
 
+If you want to build wheel with *GPU support* via CoreNEURON then we have to use image `neuronsimulator/neuron_wheel_gpu` i.e.
```

Comment:
Just thinking that we will probably converge to a single docker file. OK for a second one until we have something stable.

Reply:
yeah, I wanted to add env vars etc to choose right things inside. But 1) gpu image is huge ~14GB 2) we have to finalise setup still. So didn't to mess up existing setup that we use in production.